src/stock-user-role-scopes/add-stock-user-scope/add-stock-user-role-scope.component.tsx

```diff
@@ -421,7 +420,7 @@ const AddStockUserRoleScope: React.FC<AddStockUserRoleScopeProps> = ({ model, ed
           </section>
         </ModalBody>
```

Comment:
We shouldn't render a modal within a workspace. Modals should be rendered using the modal system. Maybe we should remove the modal related bits here and just retain the form fields. Or if the intention is to use this as a modal, then we shouldn't register it as a workspace.